alts/src/main/java/io/grpc/alts/internal/TsiHandshakeHandler.java

```diff
@@ -78,12 +82,17 @@ public abstract SecurityDetails validatePeerObject(Object peerObject)
   }
 
   private static final int HANDSHAKE_FRAME_SIZE = 1024;
+  // Avoid performing too many handshakes in parallel, as it may cause queuing in the handshake
+  // server and cause unbounded blocking on the event loop (b/168808426). This is a workaround until
```

Comment:
Drive by comment: I'd suggest removing the "b/****" in the comment. :)

Reply:
Bug numbers and CL numbers are considered safe in OSS. We don't tend to use them because they aren't helpful to other readers, though. Since this is temporary until async TSI in Q1ish and the bug details are quite involved, it seemed fair.